### ISSUE TITLE
Humble does not boot on Gaming mode.

### ISSUE THREAD

**Issue by 27hectormanuel** [Gustash/lutris_installers#10]
This actually isn't an issue with your install @Gustash.  It's a steam deck specific issue.

On the steam deck, when switching between desktop mode and gaming mode currently there is an XDG error.  Basic solution is to run:
`systemctl restart --user xdg-desktop-portal`
between switches.  Longer explanation:

https://github.com/ValveSoftware/SteamOS/issues/775

_Originally posted by @tommyfi in https://github.com/Gustash/lutris_installers/issues/9#issuecomment-1258667286_
      This solved the issue faster after I lost my previous install after a reimage of the OS but now Humble is unable to boot on Gaming mode.

**Comment by Gustash** [comment]
Yes, I have heard from other people that it may not work on Gaming Mode sometimes. My Steam Deck should arrive this week so I can try to debug this issue.

Until then I have no way to debug it as Gaming Mode does not work on Steam OS VM currently
